src/VisualStudioEditorsSetup/source.extension.vsixmanifest

```diff
@@ -4,13 +4,20 @@
     <Identity Id="F6B5EACA-7FA1-4591-8D40-A38234763621" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>VisualStudio Editors </DisplayName>
     <Description>Microsoft VisualStudio Editors</Description>
+    <ShortcutPath>..</ShortcutPath> <!-- The Final path is Common7\IDE--> 
```

Comment:
VSIX installation will create a symbolic link there that points to the random directory your extension ends up at. Is this vsix just placing binaries? or is it an actual VS extension? if so, then it might be better to be a v3.